content/docs/notebooks/setup.md

```diff
@@ -0,0 +1,251 @@
+    The image names indicate the following choices:
+
+      * A TensorFlow version (for example, `tensorflow-1.13.1`). Kubeflow offers
+        a CPU and a GPU image for each minor version of TensorFlow.
+      * `cpu` or `gpu`, depending on whether you want to train your model on a CPU 
+        or a GPU. 
+        
+          * If you choose a GPU image, make sure that you have GPUs 
+            available in your Kubeflow cluster. Run the following command to check 
+            if there are any GPUs available:
+            `kubectl get nodes "-o=custom-columns=NAME:.metadata.name,GPU:.status.allocatable.nvidia\.com/gpu"`
+          * If you have GPUs available, you can schedule your server on a GPU node 
+            in the **Extra Resources** section at the bottom of the form. For 
+            example, to reserve two GPUs, enter the following JSON code: 
+          `{"nvidia.com/gpu": 2}`
+      * Kubeflow version (for example, `v0.5.0`).
+    
+
+  * **Custom**: If you select the custom option, you must specify a Docker image 
+    in  the form `registry/image:tag`. For guidelines on creating a Docker
+    image for your notebook, see the guide to 
+    [creating a custom Jupyter image](/docs/notebooks/custom-notebook/).
+
+1. Specify the total amount of **CPU** that your notebook server should reserve. 
+  The default is `0.5`. For CPU-intensive jobs, you can choose more than one CPU 
+  (for example, `1.5`).
+
+1. Specify the total amount of **memory** (RAM) that your notebook server should 
+  reserve. The default is `1.0Gi`.
+
+1. Specify a **workspace volume** to hold your personal workspace for this
+  notebook server. Kubeflow provisions a 
+  [Kubernetes persistent volume (PV)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for your workspace volume. The PV ensures that you can
+  retain data even if you destroy your notebook server.
+
+  * The default is to create a new volume for your workspace with the
+    following configuration:
+  
+      * Name: `notebook-workspace`
+      * Size: `10Gi`
+      * Mount path: `/home/jovyan`
+      * Access mode: `ReadWriteOnce`
+
+  * Alternatively, you can point the notebook server at an existing volume by 
+    specifying the name, mount path, and access mode for the existing volume.
+
+1. *(Optional)* Specify one or more **data volumes** if you want to store and
+  access data from the notebooks on this notebook server. You can add new
+  volumes or specify existing volumes. Kubeflow provisions a 
+  [Kubernetes persistent volume (PV)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for each of your data volumes.
+
+1. Click **SPAWN** and wait a while. You should see an entry for your new
+  notebook server on the **Notebook Servers** page, with a spinning indicator in 
+  the **Status** column. It can take a few minutes to set up
+  the notebook server.
+
+    * You can check the status of your pod by entering the following command:
```

Comment:
The user can also see the status of the Pod if they hover the mouse over the Status Icon.

For example, if the image is downloading then the Status spinner will have a tooltip that says `ContainerCreating`

I think it would be nice to also include this bit of info, WDYT?

Reply:
Thanks! Done.